Program.cs

```diff
@@ -1,7 +1,58 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+
+namespace Training {
+   class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter number: ");
+         int num = int.Parse (Console.ReadLine ());
+         string words = DecimalToWords (num);
+         Console.WriteLine (words);
+         string randomNumbers = DecimalToNumbers (num);
+         Console.WriteLine (randomNumbers);
+      }
+      static string DecimalToWords (int num) {
+         string[] nums = { "", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen", "sixteen", "seventeen", "eighteen", "nineteen" };
+         string[] tens = { "", "ten", "twenty", "thirty", "forty", "fifty", "sixty", "seventy", "eighty", "ninety" };
+
+         if (num < 20) {
+            return nums[num];
+         } else if (num >= 20 && num < 100) {
+            int numDigit = num / 10;
+            int numRemainder = num % 10;
+            return tens[numDigit] + ((numRemainder == 0) ? "" : (" " + nums[numRemainder]));
+         } else if (num >= 100 && num < 1000) {
+            int hundredsDigit = num / 100;
+            int hunRemainder = num % 100;
+            string words = nums[hundredsDigit] + " hundred ";
+            if (hunRemainder != 0) {
+               words += "and ";

```

Comment:
720 shows "seven hundred **and** twenty"
1120 shows "one thousand one hundred **and** twenty"
but 1020 shows "one thousand twenty". shouldn't **and** be there?

Reply:
Done.